pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTModifierNode.java

```diff
@@ -62,10 +64,20 @@ public boolean isTransient() {
         return (node.getModifiers().getJavaModifiers() & TRANSIENT) == TRANSIENT;
     }
 
+    /**
+     * Returns true if function has `@isTest` annotation or `testmethod` modifier
+     */
     public boolean isTest() {
         return node.getModifiers().isTest();
     }
 
+    /**
+     * Returns true if function has `testmethod` modifier
+     */
+    public boolean isDeprecatedTestMethod() {
```

Comment:
yes, as the comment already says the name of this method would be better "hasDeprecatedTestMethod".... 
We can't change the name of `isTest()`, because this method already exists...

Reply:
Renamed is->has
Originally couldn't verify that this would work but now using xpath 2.0 I'm able to verify that it does.